src/Automation/Automation/Cmdlet/StartAzureAutomationDscCompilationJob.cs

```diff
@@ -25,6 +26,7 @@ namespace Microsoft.Azure.Commands.Automation.Cmdlet
     /// <summary>
     /// starts azure automation compilation job
     /// </summary>
+    [GenericBreakingChange("Behavior changed", ChangeDescription = "Cmdlet returns control to pipeline before compilation job has completed. Refer to https://docs.microsoft.com/en-us/azure/automation/automation-dsc-compile#compiling-a-dsc-configuration-with-windows-powershell")]
     [Cmdlet("Start", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "AutomationDscCompilationJob", SupportsShouldProcess = true)]
```

Comment:
This line confuses me - if it's a breaking change we can't merge into master.

Reply:
Check the issue this PR solves. Apparently, according to [documentation](https://docs.microsoft.com/en-us/azure/automation/automation-dsc-compile#compiling-a-dsc-configuration-with-windows-powershell), the intent of the cmdlet was just to start the job but not wait for its completion. However, the actual implementation waits for completion which, when the compilation job get suspended, it never ends, causing the need to just close the PS window. This change returns the cmdlet to its original intent.

Reply:
With current implementation, the documented wait-loop is useless

Reply:
@lwajswaj  if it's not a breaking change - please remove the breaking change attribute.

Reply:
@vladimir-shcherbakov , done. I've just removed the attribute